clientv1/vpp/data_change_api.go

```diff
@@ -12,18 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package defaultplugins
+package vppplugin
```

Comment:
I would argue that this might be kind of confusing to name this package `vppplugin`. Perhaps `vppapi` or `vppclient` would be more fitting since this is just some generic API for controlling VPP Plugin that inside plugins/vpp.